packages/calling-stateful-client/src/InternalCallContext.ts

```diff
@@ -51,8 +48,8 @@ export class InternalCallContext {
   // <CallId, <ParticipantKey, <StreamId, RemoteRenderInfo>>
   private _remoteRenderInfos = new Map<string, Map<string, Map<number, RemoteRenderInfo>>>();
 
-  // <CallId, LocalRenderInfo>.
-  private _localRenderInfos = new Map<string, LocalRenderInfo>();
+  // <CallId, <MediaStreamType, LocalRenderInfo>>.
+  private _localRenderInfos = new Map<string, Map<string, LocalRenderInfo>>();
 
   // Used for keeping track of rendered LocalVideoStreams that are not part of a Call.
   // The key is the stream ID. We assume each stream ID to only have one owning render info
```

Comment:
Should `_unparentedRenderInfos` now be `Map<'screenshare' | 'video', LocalRenderInfo>`?

Reply:
Yes! I will type that stronger

Reply:
updated